src/plugins/maps_dashboards/public/components/layer_control_panel/layer_control_panel.tsx

```diff
@@ -12,79 +12,150 @@ import {
   EuiListGroupItem,
   EuiButtonEmpty,
   EuiHorizontalRule,
+  EuiButton,
 } from '@elastic/eui';
 import { I18nProvider } from '@osd/i18n/react';
 import './layer_control_panel.scss';
 import { AddLayerPanel } from '../add_layer_panel';
+import { LayerConfigPanel } from '../layer_config';
+import { LAYER_TYPE } from '../../../common';
+import { ILayerConfig } from '../../model/ILayerConfig';
 
 const LayerControlPanel = () => {
+  const [isLayerConfigVisible, setIsLayerConfigVisible] = useState(false);
+  const [isLayerControlVisible, setIsLayerControlVisible] = useState(true);
+  const [selectedLayerConfig, setSelectedLayerConfig] = useState<ILayerConfig>({
+    iconType: '',
+    label: '',
+    name: '',
+    type: '',
+    id: '',
+  });
+
   // TODO: replace it once layers model ready
-  const demoLayers = [
+  const demoLayers: ILayerConfig[] = [
     {
       label: 'Base Map',
       iconType: 'visMapRegion',
+      id: 'example_id_1',
+      type: LAYER_TYPE.BASE_MAP,
+      name: 'Base Map Layer',
```

Comment:
Shall we remove Layer suffix?

Reply:
Yes, that's a example, will update when integrated with layer models